source/uwp/Renderer/lib/AdaptiveInputChoiceSet.h

```diff
@@ -34,6 +35,10 @@ namespace AdaptiveCards { namespace XamlCardRenderer
 
         IFACEMETHODIMP get_Choices(_COM_Outptr_ ABI::Windows::Foundation::Collections::IVector<ABI::AdaptiveCards::XamlCardRenderer::IAdaptiveInputChoice*>** columns);
 
+        // IAdaptiveInputElement
+        IFACEMETHODIMP get_Id(_Out_ HSTRING* text);
```

Comment:
Same for the other input classes

---
In reply to: [118370122](https://github.com/Microsoft/AdaptiveCards/pull/369#discussion_r118370122) [](ancestors = 118370122)